pynest/pynestkernel.pyx

```diff
@@ -450,7 +450,7 @@ cdef inline Datum* python_object_to_datum(obj) except NULL:
             elif numpy.issubdtype(obj.dtype, numpy.floating):
                 ret = python_buffer_to_datum[object, double](obj)
             else:
```

Comment:
```suggestion
```

Reply:
Return-early pattern: ``else`` can be removed.

Reply:
We are only storing the value in `ret` here, not returning, so I think we need the `else` statement.